pkg/tracker/tracker.go

```diff
@@ -53,7 +53,7 @@ func NewFairnessTrackerWithClockAndTicker(trackerConfig *config.FairnessTrackerC
 
 		ticker: ticker,
 
-		rotationLock: &sync.RWMutex{},
+		rotationLock: sync.RWMutex{},
```

Comment:
Usually, moving to RWMutex

You have to update code to use RLock RUnlock at some places.

Here I don't see such a thing. So I wonder why

Reply:
I didn't change the mutex to RW. It was already RWMutex. Just used original type instead of the pointer to avoid potential nil reference and bad practices mentioned in the std lib examples.

Reply:
Oh, my bad. Indeed.